features/command_line_interface/smell_selection.feature

```diff
@@ -4,15 +4,12 @@ Feature: Smell selection
   I want to be able to selectively activate smell detectors
 
   Scenario: --smell selects a smell to detect
-    When I run reek --no-line-numbers --smell DuplicateMethodCall spec/samples/two_smelly_files
+    Given a smelly file
```

Comment:
Wouldn't "Given the smelly file spec/...." be clearer? Right now this precondition is just dangling in the air...